src/main/scala/rise/autotune/configFileGeneration.scala

```diff
@@ -213,20 +165,72 @@ object configFileGeneration {
     file
   }
 
-  def valuesListToString(list: List[Any]): String = {
-    var valuesString = ""
-    list.foreach(value => {
-      valuesString += value.toString +  ", "
+  def constraintsToString(constraints: Set[Constraint]): String = {
+
+    val constraintsList = new ListBuffer[String]
+    constraints.foreach(constraint => {
+      // check type of constraint
+      val constraintString = constraint match {
+        case RangeConstraint(n, r) => {
+          val (start, stop, step) = r match {
+            case RangeAdd(start, stop, step) => (start, stop, step)
+            case RangeMul(start, stop, step) => (start, stop, step)
+            case _ => (0, 0, 0) // todo catch other types of ranges
+          }
+
+          // if stop is PosInf, remove constraint
+          // (already catched by the range of parameter)
```

Comment:
can you explain this? I am not sure I understand why removing the constraint is legal here.

Reply:
The constraints gives us the information that all values of the parameter have to be smaller than `PosInf`. We can drop this information, because it is a requirement for Hypermapper to work. Otherwise, it would not possible to generate the search space. If `stop` has a certain - finitie - value, we keep this information.

Reply:
Would it not make more sense to replace `PosInf` with `Int.MaxValue`? Or is this what Hypermapper does?

Reply:
PS: I think the comment "if stop is PosInf, remove constraint" is misleading here, I thought you meant removing the entire range constraint including start and steps bits.